src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackEncoder.cs

```diff
@@ -20,75 +23,170 @@ internal static class HPackEncoder
         //          So for now, don't do dynamic encoding.
 
         public static bool EncodeHeader(string name, string value, Span<byte> buffer, out int length)
+        {
+            Debug.Assert(value != null);
+            return EncodeHeader(name, value, null, null, buffer, out length);
+        }
+
+        public static bool EncodeHeader(Headers.HeaderDescriptor header, string value, Span<byte> buffer, out int length)
```

Comment:
It looks like these overloads (here and directly below) that take HeaderDescriptor are not used. Can we remove them?